ci-operator/step-registry/cucushift/installer/check/vsphere/folder/cucushift-installer-check-vsphere-folder-commands.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+check_result=0
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+# shellcheck disable=SC1091
+source "${SHARED_DIR}/vsphere_context.sh"
+echo "$(date -u --rfc-3339=seconds) - Configuring govc exports..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+FOLDER="/$vsphere_datacenter/vm/$vsphere_datacenter"
+SUB_FOLDER="$FOLDER/qe-folder"
+
+machine_list=$(oc get machine -n openshift-machine-api -ojson | jq -r '.items[].metadata.name')
+for machine in $machine_list; do
+    folder=$(oc get machine "$machine" -n openshift-machine-api -o=jsonpath="{.spec.providerSpec.value.workspace.folder}")
```

Comment:
the folder setting in install-config is supposed to be passed down to machine and machineset resource, I suggest to use govc command to check if the machines are really placed into the expected folder.

Reply:
What about add a govc check?

Reply:
yeah, suggest to use govc check to replace the checking of machine-api resources.